aspnetcore/migration/mvc.md

```diff
@@ -41,7 +41,7 @@ Create a new *empty* ASP.NET Core web app with the same name as the previous pro
 
 ## Configure the site to use MVC
 
-* Install the `Microsoft.AspNetCore.Mvc` and `Microsoft.AspNetCore.StaticFiles` NuGet packages.
+* By Default, the ASP.NET Core MetaPackage is added to the project, called `Microsoft.AspNetCore.All`. This package contains packages like `Microsoft.AspNetCore.Mvc` and `Microsoft.AspNetCore.StaticFiles`.
 
   `Microsoft.AspNetCore.Mvc` is the ASP.NET Core MVC framework. `Microsoft.AspNetCore.StaticFiles` is the static file handler. The ASP.NET runtime is modular, and you must explicitly opt in to serve static files (see [Work with static files](../fundamentals/static-files.md)).
```

Comment:
The ASP.NET runtime -> The ASP.NET Core runtime

Reply:
done